packages/rum/src/domain/record/node.typeGuards.ts

```diff
@@ -1,3 +1,7 @@
 export function isElement(node: Node): node is Element {
   return node.nodeType === node.ELEMENT_NODE
 }
+
+export function isShadowRoot(node: any): node is ShadowRoot {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`any` is a bit too generic here. For example, one could pass `null`, and the function would crash. What about accepting only `Node` and do some cast in the condition below?